components/canvas/drawingarea.tsx

```diff
@@ -0,0 +1,20 @@
+import styles from "./Drawing.module.css";
+import { useEffect } from "react";
+import { useRef } from "react";
```

Comment:
```suggestion
import { useEffect, useRef } from "react";
```
you only need one line in import